### ISSUE TITLE
demo.html min and max fields are not changing...

### ISSUE THREAD

**Issue by cherukumilli** [ewgenius/paper-interval-slider#3]
In the demo.html file, you have the following lines:
        min: {{$.slider.min}}<br>
        max: {{$.slider.max}}<br>
min and max don't change when we move the knob's. 

Should min and max be replaced by valueA and valueB?
        valueA: {{$.slider.valueA}}<br>
        valueB: {{$.slider.valueB}}<br>

**Comment by ewgenius** [comment]
Yes, it is my mistake, may be it is some old code, what i forget to remove
